mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -1056,16 +1056,16 @@ def coupled_ns_heat_operator(
         fluid_state, wall_kappa, wall_temperature,
         *,
         time=0.,
-        interface_noslip=True,
         use_kappa_weighted_grad_flux_in_fluid=False,
-        wall_penalty_amount=None,
         quadrature_tag=DISCR_TAG_BASE,
         fluid_gradient_numerical_flux_func=num_flux_central,
         inviscid_numerical_flux_func=inviscid_facial_flux_rusanov,
         viscous_numerical_flux_func=viscous_facial_flux_harmonic,
-        return_gradients=False):
-    r"""
-    Compute the RHS of the fluid and wall subdomains.
+        interface_noslip=True,
+        return_gradients=False,
+        wall_penalty_amount=None,
+        fluid_operator=ns_operator):
```

Comment:
```suggestion
        ns_operator=ns_operator):
```
?

Reply:
It is somewhat bothersome to me that this parameter name matches an actual function name.